src/main/resources/updater/Instruction to use past versions.txt

```diff
@@ -0,0 +1 @@
+Copy the relevant version JAR to the root folder (no need to rename it) and run that JAR file.
```

Comment:
Should this instruction be put in the root folder or inside `past_versions/`? I am concerned that putting it in `past_versions/` might hide it from users, but putting it in the root makes the root folder dirty.